.gitignore

```diff
@@ -1,3 +1,5 @@
+latest.json
```

Comment:
Ignore `latest.json` - that file is automatically generated by the script, and should usually not be shared.